### DISCUSSION TITLE
Media sync meeting 8 November 2023

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/media#67] [General]
Everyone is encouraged to add content.
Hackmd is better for realtime collab (e.g. with mutual typing or not shared screens)

We need objectives for the media group. 
> Amplify voices. How do we use the voices of the community to amplify and raise awareness of what members are doing
TODO: see what it was and if we agree?

**Comment by mrmetaverse** [comment]
Our main goals: 

1. Amplify the voices of the open metaverse 
2. Raise awareness of Open Metaverse Interoperability

**Reply by indiebio** [reply]
Create youtube playlists - started, just need to share
Retweet on the socials - ACCESS to members in progress (still need to maintain accountability)
Social media campaigns, like "Why do you love virtual worlds"
Wear different hats - can share other communities e.g. M3, and also interact with your personal handle. 
Events. e.g. Demo days, collab days. Game nights - every week, meetings

**Reply by indiebio** [reply]
Share access to indiebio and jimmy6dof for Twitter, Mastodon and youtube
